django-reference-app/django-reference-app/templates/clusterweb.html

```diff
@@ -0,0 +1,132 @@
+  <style media="print">
+    a[href^="/"]:after {
+      content: " ({{ request.scheme }}://{{ request.get_host }}" attr(href) ")";
+    }
+   
+  </style>
+{% endblock %}
+
+{% block cookie_message %}
+  <p>
+    {% trans 'GOV.UK uses cookies to make the site simpler.' %}
+    <a href="https://www.gov.uk/help/cookies">
+      {% trans 'Find out more about cookies' %}
+    </a>
+  </p>
+{% endblock %}
+
+{% block header_class %}with-proposition{% endblock %}
+{% block proposition_header %}
+  {% get_service_settings as service_settings %}
+  <div class="header-proposition">
+    <div class="content">
+      {% if service_settings.has_header_links %}
+        <a href="#proposition-links" class="js-header-toggle menu">{% trans 'Menu' %}</a>
+      {% endif %}
+      <nav id="proposition-menu">
+        <a href="{{ service_settings.header_link_url|default:'/' }}" id="proposition-name">
+          {% block proposition %}{{ service_settings.localised_name }}{% endblock %}
+        </a>
+        {% block proposition_menu %}
+          {% if service_settings.has_header_links %}
+            <ul id="proposition-links">
+              {% for link in service_settings.get_header_links %}
+                <li><a href="{{ link.url }}" class="{% if link.link_is_view_name and link.link == request.resolver_match.view_name %}active{% endif %}">{{ link.localised_name }}</a></li>
+              {% endfor %}
+            </ul>
+          {% endif %}
+        {% endblock %}
+      </nav>
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+  <main role="main" id="content" tabindex="-1">
+    {% block phase_banner %}
+      {% get_service_settings as service_settings %}
+      {% if service_settings.phase != 'live' %}
+        <div class="phase-banner">
+          <p>
+            <strong class="phase-tag">{{ service_settings.phase_name }}</strong>
+            <span>{% block phase_banner_message %}{% trans 'This is a new service.' %}{% endblock %}</span>
+          </p>
+        </div>
+      {% endif %}
+    {% endblock %}
+
+    {% block inner_content %}
+
+    <body>
+      <br>
+      <h1 style="font-size:36px;font-weight:700;">Cloud Platforms Deployment</h1>
+      <br>
+      <h2 style="font-size:24px;font-weight:400;line-height:1.25;color: #6f777b">Congratulations, You have Successfully Deployed The Kubernetes example app </h1>
+      <br>
+      <br>
+      <h2 style="font-size:24px;font-weight:600;line-height:1.25;">Getting started with the Cloud Platform</h2> 
+      <br>
+      <p>This app will give information on the new cloud platform solution</p>
+      <p>These are the follwoing repos and resources you will need to use the new platform</p>
+      <ul style="list-style:circle;padding-left:10px">
+        <li><a href="https://kubernetes.io/">Kubernetes documentation</a></li>
+        <li><a href="git@github.com:ministryofjustice/k8s-nonprod-environments.git">Non-production cluster repo</a></li>
```

Comment:
URL is https://github.com/ministryofjustice/k8s-nonprod-environments

Reply:
Yep , fair. I've amended under `25dcd79`.